python-main.js

```diff
@@ -1072,9 +1072,6 @@ function web_editor(config) {
             window.transport = new DAPjs.WebUSB(device);
             window.daplink = new DAPjs.DAPLink(window.transport);
 
-            // Ensure disconnected
-            window.daplink.disconnect();
```

Comment:
Although I think it shouldn't be possible to enter this function if already connected, perhaps we should wrap this in a try/except? Just in case.